src/main/java/dk/cngroup/lentils/exception/TeamNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package dk.cngroup.lentils.exception;
+
+public class TeamNotFoundException extends Throwable {
```

Comment:
Toto predelej takto: `public class TeamNotFoundException extends NotFoundException {`. Potom tam nebudes muset mit to try/catch, protoze NotFoundException je unchecked exception a Throwable je checked exception.

Reply:
Plus by tady pak mela byt message a zavolani super konstruktoru.